docs/resources/integration.md

```diff
@@ -34,6 +37,36 @@ resource "rollbar_integration" "slack_integration" {
     show_message_buttons = true
   }
 }
+
+# Configure the Webhook integration for the project
+#
+
+resource "rollbar_integration" "webhook_integration" {
+  webhook {
+    enabled = true
+    url = "https://www.some-website.com"
```

Comment:
some-website.com is a real URL. Can you change it to example.com?

Reply:
done